documentation/Working_on_Bot/README.md

```diff
@@ -0,0 +1,257 @@
+# Working on the Bot
+
+- [Creating Bot and Attaching it to a Development Server](https://github.com/CSSS/wall_e/wiki/2.-Creating-Bot-and-Attaching-it-to-a-Development-Server)  
```

Comment:
For the links in this list that lead to the wiki, can you indicate in brackets that they navigate away from this README? 

Also, does it make sense to have a wiki AND a README?

Reply:
>For the links in this list that lead to the wiki, can you indicate in brackets that they navigate away from this README?
fixed with 64ccf95711cd2d2e65f902c61b08a72eaf38c182

>Also, does it make sense to have a wiki AND a README?
The thinking is, things that are not changed often go into the wiki and things that may get changed often go into the wiki.

Reply:
I think it would be better to indicate that those links lead away from the current README with (links to wiki) or something like that.

Reply:
I am confused how what I did not does indicate that it leads to the wiki?

Reply:
Oh, the way you did it now is fine.